### PR TITLE
Add listenPastOnce

### PR DESCRIPTION
**Pull request by ReneB**
Well, that was surprisingly easy. I managed to do it on the ride home in public transport :smile:

### CODE REVIEW

File: SignalsTests/SignalsTests.swift [hunk 1]
```diff
@@ -140,7 +140,19 @@ class SignalsTests: XCTestCase {
...
+        emitter.onInt.fire(1)
+        emitter.onInt.fire(2)
+        listener.listenPastOnceTo(emitter)
+        emitter.onInt.fire(3)
+        emitter.onInt.fire(4)
+
+        XCTAssertEqual(listener.dispatchCount, 1, "Dispatched once")
+        XCTAssertEqual(listener.lastArgument, 2, "Remembered the most recent data")
+    }
+
```
Comment (artman) [inline SignalsTests/SignalsTests.swift]: You should add another test to check that listenPastOnce works when you add a listener before the Signal has fired once.

**Comment by artman** [comment]
Just one comment regarding adding another test, looks good otherwise.

**Comment by ReneB** [comment]
You're completely right. Fixed this oversight now :-)

**Comment by artman** [comment]
LGTM

**Comment by artman** [comment]
Thanks!

**Pull request merged**